src/components/forms/formComponents.js

```diff
@@ -175,59 +177,46 @@ export const formInputMultiText = (config) => {
   ]);
 };
 
+// Using react-select/creatable - Passing config directly through!
 export const formInputSelect = (config) => {
   const {
-    id, title, disabled, error, setError,
+    id, title, disabled, required, error, setError,
     selectOptions, placeholder, ariaDescribedby,
-    creatableConfig
+    formValue, creatable, isMulti, setFormValue,
```

Comment:
```suggestion
    formValue, setFormValue,
    isCreatable, isMulti,
```
Reorganizing the inputs so similar things are together